lib/src/lock_file.dart

```diff
@@ -219,7 +219,14 @@ class LockFile {
   /// If [entrypoint] is passed, a relative entry is added for its "lib/"
   /// directory.
   String packagesFile(SystemCache cache, [String entrypoint]) {
-    var header = 'Generated by pub on ${DateTime.now()}.';
+    var header = '''
+This file is deprecated. Tools should instead consume 
+`.dart_tools/package_config.json`.
+
+For more info see:
+https://github.com/dart-lang/language/blob/master/accepted/future-releases/language-versioning/package-config-file-v2.md
```

Comment:
How about making a `dart.dev/go/...` link? I'm not sure this URL is very stable.

Reply:
https://github.com/dart-lang/site-www/pull/2888